inspirehep/dojson/hep/fields/bd01x09x.py

```diff
@@ -33,13 +35,25 @@
 
 @hep.over('isbns', '^020..')
 @utils.for_each_value
-@utils.filter_values
 def isbns(self, key, value):
-    """Other Standard Identifier."""
-    return {
-        'value': value.get('a'),
-        'medium': value.get('b')
-    }
+    """ISBN and its medium with additional comment."""
+    isbn = {}
+    try:
+        isbn['value'] = normalize_isbn(value.get('a'))
+    except:
+        pass
+    if 'value' in isbn:
+        try:
+            isbn['medium'] = value.get('b').lower()
```

Comment:
Oh, I see. So you want to leave `medium` and `comment` blank `if 'b' not in value`?